### ISSUE TITLE
バッチごとの、ラベル別サンプル比に応じた層化抽出

### ISSUE THREAD

**Issue by F-ridge** [satsumaimokun/blindness_detection#2]
関数をつくる
torch.utils.data.dataloaderのイメージ？

入力:
dataset
batch_size: ラベルごとの数のリスト？バッチ全体のサイズ？
shuffle

出力:
イテレータ

ラベル別にdataloaderをつくってまとめるような処理でいける？

**Comment by F-ridge** [comment]
29日のミーティングまでにつくって動作とか使い方、使い心地の確認をしたい
